### ISSUE TITLE
To Celo or not to Celo?

### ISSUE THREAD

**Issue by DistributedDoge** [deora-earth/kiezDAO#3]
KiezDAO could use [Celo](https://celo.org/) blockchain so that we can take part in bonus track entitled [Peace & Prosperity](https://gitcoin.co/issue/blockchainforsocialimpact/incubator/7/4055) just skimming whitepaper:

Benefits:
- Linking identity to phone-number, verify ownership of phone number via SMS
- Stablecoins (DAI-equivalent), also fees paid in stablecoins
- Under the hood there is still EVM, so we can use solidity  contracts

Risks:
- Learning about Celo takes time
- If Celo flops we will be left without settlement layer
- I did not find security audit for the network and algorithms

In conclusion, for real-life use at this time I would rather go with bare Ethereum, but I think it might be worthwhile to prototype some functionality using Celo (especially due to phone<->identity link) and see how user/developer experience compares to `typical` dApp.

Since stakes are low, and deadline is far away, I signed up for the contest track on gitcoin, as we can always change our minds later.

If you have any opinions on **why Celo** or **why not Celo**, I would be interested in hearing them.

**Comment by DistributedDoge** [comment]
Notes from *Celo* webinar regarding philosophy behind the platform:

- Plans for security audit from OpenZeppelin, nothing public as of yet.
- No main-net either, just test-nets. 
- Celo mainly thinks of `unbanked` who also have `no formal ID` idea is to create open financial system usable by anyone with `mobile phone` (as long as it is `smart-phone`), stress on `mobile first`. Example is the `refugee` (...very common cause, but see Q&A for problems). 
- Celo thinks Ethereum UX is bad for mobile, hence need for Celo (selling points: Stable Value assets Managed By Network Itself, phone number as identity, gas payable in multiple currencies)

Q&A session:
- "They can't do cross-chain transactions as of yet, but should be easy to do."
- Interoperability between blockchains is a priority, "Celo is a cousin of Ethereum". 
- Roast from Consensys employee about security/speed trade-off (claim was Celo more-centralized and less-secure as it combines short time between blocks with Delegated Proof-of-Stake)
- Migrants and refugges switch SIM cards as they arrive in new country, thus undermining core assumption and making number-based identity a bit tricky. 
- They target smartphones, so not really interested in m-pesa style SMS-only transfers.
- Funds are `really` managed by private private key stored on phone. Phone number is just means of addressing pub_key when sending funds.
- Exchanges, applications built on top of celo should think of  AML/KYC 
- Design `with communities` not `for them` approach, there is a blog on medium.

**Comment by johannbarbie** [comment]
cool, i can start by using their contract kit in the unit tests, and cUSD for borrowing and collateral: https://repl.it/@celoOrg/contractKit

**Comment by DistributedDoge** [comment]
Note for my future biographers: We decided to give Celo a shot after all. Closing.
